.github/workflows/deploy-release.yml

```diff
@@ -23,17 +25,75 @@ jobs:
           secrets: ${{ toJSON(secrets) }}
           prod: ${{ true }}
 
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v5
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+          git_tag_gpgsign: true
+
+      - name: Create archive
+        run: tar -czf "$ARCHIVE_NAME".tar.gz out
+
+      - name: Create checksum
+        run: sha256sum "$ARCHIVE_NAME".tar.gz > ${{ env.ARCHIVE_NAME }}-sha256-checksum.txt
+
+      # Upload the checksum file so that it can be reused in the next job
+      - uses: actions/upload-artifact@v2
+        with:
+          name: checksum-file
+          path: ${{ env.ARCHIVE_NAME }}-sha256-checksum.txt
```

Comment:
Can't it be uploaded straight as a release asset? I don't see it used for anything else.